### ISSUE TITLE
Regular users cannot submit BBC or HTML articles

### ISSUE THREAD

**Issue by Bopske** [Tinyportal/TinyPortal#461]
When testing it on 2.0.0 as a simple user that should be able to submit articles I found some funny issues:
- the menu options submit HTML article and submit BBC articles are showing, but clicking them does not bring the user to the create article screen
(this works fine for an admin)

- I change the article author for an existing article to the test user, then when accessing the article via the edit button as the test user I get this error:
Your session timed out while posting. Please go back and try again.

In short: the edit article functionality for non-admins seems to be a bit broken in 2.0.0 atm...
